client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -354,11 +354,14 @@ const SyncCardContainer = ( {
 							status="is-error"
 							icon="mention"
 							showDismiss={ false }
-							text={ translate( 'We couldn’t synchronize changes to the %(siteType)s site.', {
-								args: {
-									siteType: siteToSync,
-								},
-							} ) }
+							text={ translate(
+								'We couldn’t synchronize changes to the %(siteType)s site and will need to manually diagnose the issue. Please contact support.',
```

Comment:
I would change the message to something like: 'We couldn’t synchronize changes to the %(siteType)s sit.  Please contact support for next steps.`

I am not sure that we need to specify that we would need to manually diagnose the issue - I think it is already somewhat clear from contacting support part that the resolution would take extra steps.

Reply:
Thanks @katinthehatsite. @danielbachhuber @javierarce  and thoughts on wording? Now it's:

> We couldn’t synchronize changes to the %(siteType)s site.

And current options:

> We couldn’t synchronize changes to the %(siteType)s site and will need to manually diagnose the issue. Please contact support.
> We couldn’t synchronize changes to the %(siteType)s site. Please contact support for the next steps.

Reply:
Another option:
>  We couldn’t synchronize changes to the %(siteType)s site. Please contact support for assistance diagnosing the issue.

I don't have a strong preference, though.

Reply:
I'd just say: "We couldn’t synchronize changes to the %(siteType)s site. Please contact support."